CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Minor: The account switcher is now styled to match your theme. (#4817)
 - Minor: Add an invisible resize handle to the bottom of frameless user info popups and reply thread popups. (#4795)
 - Minor: The installer now checks for the VC Runtime version and shows more info when it's outdated. (#4847)
+- Minor: Added missing file `Details` properties for Windows builds. (#4912)
```

Comment:
Make this a `dev` entry and don't phrase it as adding something that's missing, phrase it as it being added as a feature instead

Reply:
done e95e606c6cfe46452696b76dc73a31b06ff61655